run.sh

```diff
@@ -1,20 +1,64 @@
 #!/bin/bash
 
-access_id=${SUMO_ACCESS_ID:=$1}
-access_key=${SUMO_ACCESS_KEY:=$2}
-receiver_url=${SUMO_RECEIVER_URL:=https://collectors.sumologic.com}
-collector_name=${SUMO_COLLECTOR_NAME:=collector_container}
-sources_json=${SUMO_SOURCES_JSON:=/etc/sumo-sources.json}
+SUMO_ACCESS_ID=${SUMO_ACCESS_ID:=$1}
+SUMO_ACCESS_KEY=${SUMO_ACCESS_KEY:=$2}
+SUMO_RECEIVER_URL=${SUMO_RECEIVER_URL:=https://collectors.sumologic.com}
+SUMO_COLLECTOR_NAME=${SUMO_COLLECTOR_NAME_PREFIX:=''}${SUMO_COLLECTOR_NAME:=`cat /etc/hostname`}
```

Comment:
That's correct for linux but you can't change the hostname within a docker container. So even if you update/replace /etc/hostname it won't actually be updated if you run the hostname command. This is why i call `cat /etc/hostname` instead of the collector (as you can't override it)